src/routes/profil/+page.ts

```diff
@@ -0,0 +1,27 @@
+import { error } from '@sveltejs/kit';
+
+import type { PageServerLoad } from './$types';
```

Comment:
`PageServerLoad` is the type for `+page.server.ts` `load` function.

For `+page.ts`, I think it should be `PageLoad`.